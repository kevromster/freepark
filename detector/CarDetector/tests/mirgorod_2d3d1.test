config = camera_nsk_mirgorod_2d3d.txt
input = input/mirgorod_2d3d1.png
expected = expected/mirgorod_2d3d1.out
