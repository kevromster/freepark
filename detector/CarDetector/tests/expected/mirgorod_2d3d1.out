[
parking id: 26
spots count: 3

{ lat: 54.85369; lon: 83.039321; complexity: NEWBIE }
{ lat: 54.853704; lon: 83.03938; complexity: NEWBIE }
{ lat: 54.853719; lon: 83.03944; complexity: NEWBIE }
]

